src/core/loop_induction_variables/include/noelle/core/InductionVariable.hpp

```diff
@@ -58,6 +77,8 @@ class InductionVariable {
 
   PHINode *getLoopEntryPHI(void) const;
 
+  PHINode *getSCEVPHI(void) const;
```

Comment:
What is the difference between the loop entry PHI and the SCEVPHI?

Reply:
the SCEVPHI is the PHINode whose SCEV we want to use for IV properties such as the step size. the loopEntryPHI is the entry PHINode to the loop associated with the IV. these two PHINodes differ when we want to catch an IV of the outer loop but it is only being updated in the inner loop, the pattern this pull request addresses.